bootstrap/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker:17.03
+FROM docker:17.03.1-ce
```

Comment:
Thanks @ndegory, changes LGTM and I'm preparing the complementary PR right now to support this. I already updated our DinD images to 17.05-ce-rc1 and [17.05-ce-rc2 for amptools](https://github.com/appcelerator/amp/blob/master/images/amptools/Dockerfile#L11) (although we should probably downgrade that to rc1 for now) ... so maybe we should bump this up to rc1 as well (for consistency)?